code/__DEFINES/DNA.dm

```diff
@@ -196,6 +196,30 @@
 #define ORGAN_SLOT_XENO_PLASMAVESSEL "plasma_vessel"
 #define ORGAN_SLOT_XENO_RESINSPINNER "resin_spinner"
 
+/// Tumor slots - cancer but needed for compatibility
+/// External
+#define ORGAN_SLOT_ETUMOR1 "etumor1"
+#define ORGAN_SLOT_ETUMOR2 "etumor2"
+#define ORGAN_SLOT_ETUMOR3 "etumor3"
+#define ORGAN_SLOT_ETUMOR4 "etumor4"
+#define ORGAN_SLOT_ETUMOR5 "etumor5"
+#define ORGAN_SLOT_ETUMOR6 "etumor6"
```

Comment:
so many slots needed really shows the downside of the way the organ system is coded, I should get back on my rework